src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotUseDeprecatedSecurityProtocolsTests.cs

```diff
@@ -435,14 +435,14 @@ public void TestMethod()
         ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | (SecurityProtocolType)192;
     }
 }",
-                GetCSharpResultAt(9, 48, DoNotUseDeprecatedSecurityProtocols.HardCodedRule, "Tls12"),
-                GetCSharpResultAt(9, 48, DoNotUseDeprecatedSecurityProtocols.DeprecatedRule, "3264"));
+                VerifyCS.Diagnostic(DoNotUseDeprecatedSecurityProtocols.DeprecatedRule).WithSpan(9, 48, 9, 102).WithArguments("3264"),
+                VerifyCS.Diagnostic(DoNotUseDeprecatedSecurityProtocols.HardCodedRule).WithSpan(9, 48, 9, 74).WithArguments("Tls12"));
         }
 
         [Fact]
-        public void TestUse768DeconstructionAssignmentNoDiagnostic()
+        public async Task TestUse768DeconstructionAssignmentNoDiagnostic()
```

Comment:
Test is failing with the following error

```// Test0.cs(10,53): error CS8356: Predefined type 'System.ValueTuple`2' is declared in multiple referenced assemblies: 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089' and 'System.ValueTuple, Version=4.0.1.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'
```

I am pretty sure that I screwed up something in the `Test` class definition mixing netfwk and netstandard or something like that but I cannot really spot where is the mistake.

Reply:
My wild guess is `AdditionalMetadataReferences.Default` including the Microsoft.CodeAnalysis package. One work around is having a separate set of ReferenceAssemblies for the Security*CodeFixVerifiers that doesn't include the Microsoft.CodeAnalysis package.

Maybe someone has a better idea? 🙂

Reply:
It solves the issue but now all the tests using `Microsoft.CodeAnalysis` fail. I will go this way and update locally waiting for a better option.

Reply:
On the second though that doesn't seem right to go this way. I'll wait for someone input here. @sharwell, @mavasani any idea?

Reply:
Yeah, seems like a bit of a hack to me. Given that this is a large PR and also blocks the corresponding PR in master, I think it is reasonable to create tracking issues for any workarounds that can be put to unblock this PR. Lets get this PR merged soon and try to tackle the correct fix with a follow-up PR.

Reply:
See #3092